src/test/java/proxy/HelloTargetCallbackFilter.java

```diff
@@ -0,0 +1,16 @@
+package proxy;
+
+import net.sf.cglib.proxy.CallbackFilter;
+
+import java.lang.reflect.Method;
+
+public class HelloTargetCallbackFilter implements CallbackFilter {
+    @Override
+    public int accept(Method method) {
+        if (method.getName().equals("sayHello")) {
+            return 0;
+        } else {
+            return 1;
+        }
```

Comment:
```suggestion
            return method.getName().equals("sayHello");
```

객체지향 생활 체조 원칙 (참고만)

규칙 2: else 예약어를 쓰지 않는다.